content/doc/book/installing/_docker-for-tutorials.adoc

```diff
@@ -97,7 +97,7 @@ RUN add-apt-repository \
        $(lsb_release -cs) stable"
 RUN apt-get update && apt-get install -y docker-ce-cli
 USER jenkins
-RUN jenkins-plugin-cli --plugins blueocean:1.24.2
+RUN jenkins-plugin-cli --plugins blueocean:1.24.3
```

Comment:
Do you need a version here?

Reply:
> Do you need a version here?

We're intentionally trying to model the recommended practice of specifying a version number so that the configuration does not change from one build to the next build.  

I've wrestled with the "do you need a version here?" question because this page would be best if it automatically used the most recent release of the Jenkins LTS and the most recent release of the Blue Ocean plugin.  However, haven't yet spent the time to find a way to extend the "most recent LTS" concept from haml files into adoc.

@vsilverman and I have discussed back and forth on the topic and settled on this for now.